File: Trab TADSS/cs/Services/ExportadorCsv.cs
using System.Collections.Generic;
using System.Globalization;
using System.IO;
using System.Text;
using SimuladorFilas.Models;

namespace SimuladorFilas.Services
{
    public static class ExportadorCsv
    {
        public static void Exportar(string caminho, IEnumerable<ResultadoCenario> resultados)
        {
            var sb = new StringBuilder();

            sb.AppendLine("Atendentes,Repeticoes,MediaClientesGerados,MediaThroughput,MediaTempoEsperaFila,IC95InferiorEsperaFila,IC95SuperiorEsperaFila,MaxTempoEsperaFilaObservado,MediaTempoAtendimento,MaxTempoAtendimentoObservado,MediaLeadTime,MediaPercentualClientesDentroMeta,CumpreMeta");

            foreach (var resultado in resultados)
            {
                sb.AppendLine(string.Join(",",
                    resultado.Atendentes,
                    resultado.Repeticoes,
                    resultado.MediaClientesGerados.ToString("F2", CultureInfo.InvariantCulture),
                    resultado.MediaThroughput.ToString("F2", CultureInfo.InvariantCulture),
                    resultado.MediaTempoEsperaFila.ToString("F2", CultureInfo.InvariantCulture),
                    resultado.IC95InferiorEsperaFila.ToString("F2", CultureInfo.InvariantCulture),
                    resultado.IC95SuperiorEsperaFila.ToString("F2", CultureInfo.InvariantCulture),
                    resultado.MaxTempoEsperaFilaObservado.ToString("F2", CultureInfo.InvariantCulture),
                    resultado.MediaTempoAtendimento.ToString("F2", CultureInfo.InvariantCulture),
                    resultado.MaxTempoAtendimentoObservado.ToString("F2", CultureInfo.InvariantCulture),
                    resultado.MediaLeadTime.ToString("F2", CultureInfo.InvariantCulture),
                    resultado.MediaPercentualClientesDentroMeta.ToString("F2", CultureInfo.InvariantCulture),
                    resultado.CumpreMeta ? "SIM" : "NAO"
                ));
            }

            File.WriteAllText(caminho, sb.ToString(), Encoding.UTF8);
        }
    }
}

File: Trab TADSS/java/src/services/ExportadorCsv.java
package src.services;

import src.models.ResultadoCenario;

import java.io.BufferedWriter;
import java.io.FileWriter;
import java.io.IOException;
import java.util.List;
import java.util.Locale;

public class ExportadorCsv {

    public static void exportar(String caminho, List<ResultadoCenario> resultados) {
        try (BufferedWriter writer = new BufferedWriter(new FileWriter(caminho))) {
            writer.write("Atendentes,Repeticoes,MediaClientesGerados,MediaThroughput,MediaTempoEsperaFila,IC95InferiorEsperaFila,IC95SuperiorEsperaFila,MaxTempoEsperaFilaObservado,MediaTempoAtendimento,MaxTempoAtendimentoObservado,MediaLeadTime,MediaPercentualClientesDentroMeta,CumpreMeta");
            writer.newLine();

            for (ResultadoCenario resultado : resultados) {
                writer.write(String.join(",",
                        String.valueOf(resultado.getAtendentes()),
                        String.valueOf(resultado.getRepeticoes()),
                        String.format(Locale.US, "%.2f", resultado.getMediaClientesGerados()),
                        String.format(Locale.US, "%.2f", resultado.getMediaThroughput()),
                        String.format(Locale.US, "%.2f", resultado.getMediaTempoEsperaFila()),
                        String.format(Locale.US, "%.2f", resultado.getIc95InferiorEsperaFila()),
                        String.format(Locale.US, "%.2f", resultado.getIc95SuperiorEsperaFila()),
                        String.format(Locale.US, "%.2f", resultado.getMaxTempoEsperaFilaObservado()),
                        String.format(Locale.US, "%.2f", resultado.getMediaTempoAtendimento()),
                        String.format(Locale.US, "%.2f", resultado.getMaxTempoAtendimentoObservado()),
                        String.format(Locale.US, "%.2f", resultado.getMediaLeadTime()),
                        String.format(Locale.US, "%.2f", resultado.getMediaPercentualClientesDentroMeta()),
                        resultado.isCumpreMeta() ? "SIM" : "NAO"
                ));
                writer.newLine();
            }
        } catch (IOException e) {
            throw new RuntimeException("Erro ao exportar CSV", e);
        }
    }
}